lib/cocina/models/mapping/to_mods/title.rb

```diff
@@ -178,7 +178,7 @@ def write_structured(title:, title_info_attrs: {})
               title_parts_without_names.each do |title_part|
                 title_type = tag_name_for(title_part)
                 title_value = title_value_for(title_part, title_type, title)
-                xml.public_send(title_type, title_value) if title_part.note.blank?
+                xml.public_send(title_type, title_value) if title_part.note.blank? && title_type
```

Comment:
this might be better addressed by putting a title in the pub_year spec in dor_indexing_app

Reply:
we fixed the problem in the dor_indexing_app specs.